core/eth/contracts/FluxAggregator.go

```diff
@@ -0,0 +1,181 @@
+package contracts
+
+import (
```

Comment:
Why the break from standard Go convention for naming files>

Reply:
Took a cue from `abigen` here, which names the .go file identically to the Solidity contract name.